src/assets.ts

```diff
@@ -46,10 +46,9 @@ export interface AssetSource {
 }
 
 export async function getAsset(filePath: string): Promise<Asset | undefined> {
+  const videoConfig = await getVideoConfig();
   const assetConfigPath = await getAssetConfigPath(filePath);
-  const file = await readFile(assetConfigPath);
-  const asset = JSON.parse(file.toString());
-  return asset;
+  return videoConfig.loadAsset(assetConfigPath)
```

Comment:
(blocking) I think we want the argument here for `loadAsset` (and the other new hooks) to be the `filePath` variable because we're doing a bunch of things in `getAssetConfigPath` that are dependent on videoConfig settings. 

Otherwise devs would have to import the `getAssetConfigPath` function.

Reply:
my bad, for some reason I was thinking someone had to provide this assetPath but it's just given to the callback.